src/main/resources/jenkins/plugins/zulip/ZulipNotifier/global.jelly

```diff
@@ -29,6 +29,15 @@
     <f:entry title="Default Topic Name" help="/plugin/zulip/help-globalConfig-topic.html">
         <f:textbox name="topic" value="${descriptor.getTopic()}" />
     </f:entry>
+    <f:entry title="Use full job path in default topic name" help="/plugin/zulip/help-globalConfig-fullJobPathAsDefaultTopic.html">
+        <f:checkbox name="fullJobPathAsDefaultTopic" checked="${descriptor.isFullJobPathAsDefaultTopic()}" />
+    </f:entry>
+    <f:entry title="Use full job path in notification message" help="/plugin/zulip/help-globalConfig-fullJobPathInMessage.html">
+        <f:checkbox name="fullJobPathInMessage" checked="${descriptor.isFullJobPathInMessage()}" />
+    </f:entry>
+    <f:entry title="Enable Smart Notification" help="/plugin/zulip/help-globalConfig-smartNotify.html">
```

Comment:
Duplicates the smart notification settings.

Reply:
Sorry, that was a copy/paste error. I'll remove it.